client/reports.go

```diff
@@ -42,6 +43,7 @@ func (c *Client) staticSendReport(gcas GCAServer, er EnergyRecord) {
 	eqr.Signature = glow.Sign(sb, c.staticPrivKey)
 	data := eqr.Serialize()
 	location := fmt.Sprintf("%v:%v", gcas.Location, gcas.UdpPort)
+	c.EventLog.Printf("udp send ShortID %v Timeslot %v PowerOutput %v", eqr.ShortID, eqr.Timeslot, eqr.PowerOutput)
```

Comment:
I know this only happens every 5 minutes, but do you think that might be too spammy? My fear is that an operator would miss a real error because it's buried in a bunch of udp send logs.

Instead maybe we can have a variable that tracks the most recent udp send, and the most recent non-zero udp send and include those values in the larger dump rather than in the event log.

Reply:
While we're at it, we should probably also have a variable that tracks the most recent successful sync operation. Just in case the sync information is buried in other errors, it'd be nice to have somewhere to look and see if the client thinks that it's syncing successfully.

Reply:
These are good ideas. I resisted this approach, however I will do a lookover and create some statistics to reduce the need for logs except when they are useful to find problems.